pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -1167,7 +1167,7 @@ https://github.com/junegunn/vim-plug/,,
 https://github.com/powerman/vim-plugin-AnsiEsc/,,
 https://github.com/hasundue/vim-pluto/,HEAD,
 https://github.com/sheerun/vim-polyglot/,,
-https://github.com/jakwings/vim-pony/,,
+https://github.com/jmcomets/vim-pony/,,
```

Comment:
That looks like a different plugin

https://web.archive.org/web/20220914015233/https://github.com/jakwings/vim-pony

Seems like 9bf526ee410377838431d5541e7c50572054dbc4 doesn't need the update to work, so maybe the updates can just be removed?

Reply:
Ok ! I removed all commits except the relevant one :)